File: python/01-commands/doc/Banking_Kara/account_statement_print.py
def print_statement(transaction_list):
    printed_statement = "Date Amount Balance\n"
    for transaction in transaction_list:
        printed_statement += f"{transaction.date.strftime('%m/%d/%Y')} " \
                             f"{'+' if transaction.amount >= 0 else ''}{transaction.amount} " \
                             f"{transaction.balance}\n"
    return printed_statement

File: java/01-commands/doc/sourcecode/src/main/java/com/training/app/AccountStatementPrint.java
package com.training.app;

import java.util.List;

public class AccountStatementPrint {
    public static String printStatement(List<BankingTransactionDTO> transactionList) {
        String printedStatement = "Date Amount Balance\n";
        for (BankingTransactionDTO transaction : transactionList) {
            printedStatement += transaction.getDate().toString() + " " +
                    (transaction.getAmount() >= 0 ? "+" : "") + 
                    transaction.getAmount() + " " +
                    transaction.getBalance() + "\n";
        }

        return printedStatement;
    }
}
